java/src/org/openqa/selenium/chromium/ChromiumOptions.java

```diff
@@ -274,7 +274,46 @@ protected Object getExtraCapability(String capabilityName) {
   protected void mergeInPlace(Capabilities capabilities) {
     Require.nonNull("Capabilities to merge", capabilities);
 
-    capabilities.getCapabilityNames().forEach(name -> setCapability(name, capabilities.getCapability(name)));
+    capabilities.getCapabilityNames().forEach(name -> {
```

Comment:
Maybe we can use the traditional `for..each` because this lambda that we are passing into `forEach()` is multi lined. The conventional `for..each` helps with readability.

Reply:
I appreciate the concern. However, even after making changes, I did not see an improvement since the loop logic will be in a code block in the next line and the defining the loop remains in the line above. So I think it is okay to keep it as it is.

Reply:
I was not suggesting the use of a traditional for loop to improve performance, but its a convention that is followed when dealing with lamdas (atleast that's what I have learnt from the sessions that I have watched run by Venkat Subramanian) that lamdas should be a single line, if they can't be, then they perhaps are not lambdas.

Reply:
I understand, I was talking about readability itself. I don't see any improvement in how it looks but maybe I am missing something. This is something new I learned today. Thank you!